dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java

```diff
@@ -332,272 +283,304 @@
 
-    public static final String REGISTRY_SEPARATOR = "|";
+    public static final String REFERENCE_FILTER_KEY = "reference.filter";
 
-    public static final Pattern REGISTRY_SPLIT_PATTERN = Pattern
-            .compile("\\s*[|;]+\\s*");
+    public static final String INVOKER_LISTENER_KEY = "invoker.listener";
 
-    public static final String SEMICOLON_SEPARATOR = ";";
+    public static final String SERVICE_FILTER_KEY = "service.filter";
 
-    public static final Pattern SEMICOLON_SPLIT_PATTERN = Pattern
-            .compile("\\s*[;]+\\s*");
+    public static final String EXPORTER_LISTENER_KEY = "exporter.listener";
 
-    public static final String CONNECT_QUEUE_CAPACITY = "connect.queue.capacity";
+    public static final String ACCESS_LOG_KEY = "accesslog";
 
-    public static final String CONNECT_QUEUE_WARNING_SIZE = "connect.queue.warning.size";
+    public static final String ACTIVES_KEY = "actives";
 
-    public static final int DEFAULT_CONNECT_QUEUE_WARNING_SIZE = 1000;
+    public static final String CONNECTIONS_KEY = "connections";
 
-    public static final String CHANNEL_ATTRIBUTE_READONLY_KEY = "channel.readonly";
+    public static final String ID_KEY = "id";
 
-    public static final String CHANNEL_READONLYEVENT_SENT_KEY = "channel.readonly.sent";
+    public static final String ASYNC_KEY = "async";
 
-    public static final String CHANNEL_SEND_READONLYEVENT_KEY = "channel.readonly.send";
+    public static final String FUTURE_GENERATED_KEY = "future_generated";
 
-    public static final String COUNT_PROTOCOL = "count";
+    public static final String FUTURE_RETURNTYPE_KEY = "future_returntype";
 
-    public static final String TRACE_PROTOCOL = "trace";
+    public static final String RETURN_KEY = "return";
 
-    public static final String EMPTY_PROTOCOL = "empty";
+    public static final String TOKEN_KEY = "token";
 
-    public static final String ADMIN_PROTOCOL = "admin";
+    public static final String INTERFACES = "interfaces";
 
-    public static final String PROVIDER_PROTOCOL = "provider";
+    public static final String GENERIC_KEY = "generic";
 
-    public static final String CONSUMER_PROTOCOL = "consumer";
+    public static final String LOCAL_PROTOCOL = "injvm";
+    // END dubbo-rpc-api
 
-    public static final String ROUTE_PROTOCOL = "route";
 
-    public static final String SCRIPT_PROTOCOL = "script";
+    // BEGIN dubbo-rpc-rest
+    public static final String KEEP_ALIVE_KEY = "keepalive";
 
-    public static final String CONDITION_PROTOCOL = "condition";
+    public static final boolean DEFAULT_KEEP_ALIVE = true;
 
-    public static final String MOCK_PROTOCOL = "mock";
+    public static final String EXTENSION_KEY = "extension";
+    // END dubbo-rpc-rest
 
-    public static final String RETURN_PREFIX = "return ";
 
-    public static final String THROW_PREFIX = "throw";
+    // BEGIN dubbo-config-api
+    public static final String CLUSTER_KEY = "cluster";
```

Comment:
This suggestion is put on dubbo-cluster

Reply:
hesitate to move it into dubbo-cluster. It's only used in dubbo-config-api.